meteor/server/api/translationsBundles.ts

```diff
@@ -0,0 +1,46 @@
+import { TranslationsBundles, TranslationsBundleId } from '../../lib/collections/TranslationsBundles'
+import { TranslationsBundle, TranslationsBundleType } from 'tv-automation-sofie-blueprints-integration'
+import { getRandomId, unprotectString } from '../../lib/lib'
+import { logger } from '../logging'
+import { BlueprintId } from '../../lib/collections/Blueprints'
+
+export function upsertBundles(bundles: TranslationsBundle[], parentBlueprintId: BlueprintId) {
+	for (const bundle of bundles) {
+		const { type, language, data } = bundle
+
+		if (type !== TranslationsBundleType.I18NEXT) {
+			throw new Error(`Unknown bundle type ${type}`)
+		}
+
+		const namespace = unprotectString(parentBlueprintId)
+		const _id = getExistingId(namespace, language) || getRandomId<'TranslationsBundleId'>()
+
+		TranslationsBundles.upsert(
+			_id,
+			{ _id, type, namespace, language, data },
+			{ multi: false },
+			(
```

Comment:
Is there a reason for using the callback pattern here?
If there is an error it wont be displayed to the user, which might be unhelpful.